ibm/resource_ibm_container_cluster.go

```diff
@@ -396,7 +396,7 @@ func resourceIBMContainerCluster() *schema.Resource {
 				Type:        schema.TypeSet,
 				Optional:    true,
 				Computed:    true,
-				Elem:        &schema.Schema{Type: schema.TypeString},
+				Elem:        &schema.Schema{Type: schema.TypeString, ValidateFunc: InvokeValidator("ibm_is_vpc", "tag")},
```

Comment:
I think here you cann't use ibm_is_vpc in all these we need to add the resourceInvoke validator and add each respective resource name